.pre-commit-hooks.yaml

```
@@ -13,7 +13,7 @@
     Rewrites all Terraform configuration files to a canonical format.
   entry: hooks/terraform_fmt.sh
   language: script
-  files: \.(tf|tofu|tfvars)$
+  files: \.(tf|tofu|tfvars|tftest\.hcl|tfmock\.hcl)$
```

Optimize regex (may worsen readability though):
```suggestion
  files: \.(tf(vars|(mock|test)\.hcl)?|tofu)$
```

---

Let's leave as is for readability